gallery/src/styles/NxTile/NxTilePage.tsx

```diff
@@ -46,20 +47,36 @@ const NxTilePage = () =>
             <td className="nx-cell">Nested inside <code className="nx-code">.nx-tile</code></td>
             <td className="nx-cell">Used for tile titles, it has title and optional sub-title elements.</td>
           </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-tile-header__headings</code></td>
+            <td className="nx-cell"><code className="nx-code">NxTile.Headings</code></td>
+            <td className="nx-cell">Nested inside <code className="nx-code">.nx-tile-header</code></td>
+            <td className="nx-cell">
+              If there is a sub-title then the <NxCode>h2.nx-tile-header__title</NxCode> &amp;
+              {' '}<NxCode>h3.nx-tile-header__subtitle</NxCode> should both be wrapped in a containing
+              {' '}<code className="nx-code">&lt;hgroup&gt;</code> with this class. If there
+              is only an <code className="nx-code">.nx-tile-header__title</code> then this element and its class are
+              optional.
+            </td>
+          </tr>
```

Comment:
Should probably use the convenience components everywhere

Reply:
> Should probably use the convenience components everywhere

This page is covered in the iDay PR.

Reply:
> This page is covered in the iDay PR.

That's my point.  These new lines _aren't_ covered in that PR

Reply:
> > This page is covered in the iDay PR.
> 
> That's my point. These new lines _aren't_ covered in that PR

Ah yes, that could be an ugly merge conflict. Converted here: https://github.com/sonatype/sonatype-react-shared-components/pull/457/commits/2e97faac81405f7c6c05143d9047fab1a8abddab